Tests/Configuration/ConfigurationBuilderTest.php

```diff
@@ -237,7 +234,7 @@ private function setAssetsHelperMock($filename)
             ->with($this->equalTo(''))
             ->will($this->returnValue($filename));
 
-        $this->container->set('templating.helper.assets', $assetsHelper, 'request');
+        $this->container->setParameter('templating.helper.assets', $assetsHelper, 'request');
```

Comment:
Same here, this needs to stay as a service and not a parameter. The last argument can be dropped though